webpack/webpack.config.prod.js

```diff
@@ -2,19 +2,22 @@ var path = require('path');
 var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var InlineEnviromentVariablesPlugin = require('inline-environment-variables-webpack-plugin');
+var fs = require('fs');
 
 var commonLoaders = require('./common.config').commonLoaders;
 var assetsPath = require('./common.config').assetsPath;
 var publicPath = require('./common.config').publicPath;
 
-commonLoaders = commonLoaders.concat(
-  {
-    test: /\.css$/,
-    loader: ExtractTextPlugin.extract('style-loader', 'css-loader?module!postcss-loader')
-  }
-);
+var nodeModules = fs.readdirSync('node_modules')
+                    .filter(function(x) {
+                      return ['.bit'].indexOf(x) === -1;
+                    })
+                    .reduce(function(acc, cur) {
+                      acc[cur] = 'commonjs ' + cur;
+                      return acc;
+                    }, {});
```

Comment:
`nodeModules` here is used to specify `externals`, perhaps we should name it `externals`?

I based this PR off this post by James Long from a while back - http://jlongster.com/Backend-Apps-with-Webpack--Part-I. 

Basically on the server we do not want to be bundling `node_modules` into our resulting bundle, because of how some `node_modules` use `require`. However, we can't just specify node_modules as externals, because webpack `externals` assumes a browser environment, and will just turn `require("foo")` is into just `foo`, a global variable. We want to keep the `require`, so we need to prefix the value with `commonjs `. So the result of this is a `server.js` bundle that has external dependencies - as opposed to a bundle that has already bundled in the `node_modules`.

Apart from that article, [this](https://webpack.github.io/docs/configuration.html#externals) section in the webpack docs also specifies how the `commonjs` prefix gets treated.